docs/modules/ROOT/pages/tmail-backend/run.adoc

```diff
@@ -49,6 +56,17 @@ docker run -d --network emaily -e "JAVA_TOOL_OPTIONS=-Xmx500m -Xms500m" --hostna
 link:https://github.com/linagora/tmail-backend/blob/master/tmail-backend/apps/distributed/docker-compose.yml[There is a docker-compose file]
 you can use to run everything in the same network at once.
 
+First you need to add your JWT keys to the docker-compose file, like this:
```

Comment:
Instead can we add the volumes to the docker-compose (target file don't exist) and explain here, and in a comment of the docker-compose how to generate them?

Reply:
Better give more details than not enough.